Sources/ParseSwift/Types/ParseGeoPoint.swift

```diff
@@ -154,3 +165,24 @@ extension ParseGeoPoint: CustomStringConvertible {
         debugDescription
     }
 }
+
+// MARK: Convenience
+extension ParseGeoPoint {
+    /**
+     Creates a new `CLLocation` instance for the given `ParseGeoPoint`, set to the location's coordinates.
+     - parameter geopoint: Instance of `ParseGeoPoint`, with set latitude and longitude.
+     - returns: Returns a `CLLocation`
+     */
+    public func toCLLocation() -> CLLocation {
+        return CLLocation(latitude: latitude, longitude: longitude)
```

Comment:
You can remove the keyword “return” on one liner methods

Reply:
ok